desktop/apps/artwork/components/artists/index.jade

```diff
@@ -2,10 +2,10 @@ include ../../../../components/side_tabs/mixins
 
 if artwork.artists.length
   for artist in artwork.artists
-    - var tabs = helpers.artists.build(artist)
+    - var tabs = helpers.artists.build(artist, isAdmin)
 
     if tabs.length
-      section.artwork-section.artwork-artist
+      section.artwork-section.artwork-artist(data-artist-id=artist._id)
```

Comment:
We render the artist sections server-side, and then client-side we want to 'find' those div's again and be able to access the artist id of each one, so store it in the DOM.